netci.groovy

```diff
@@ -744,14 +747,16 @@ def static getJobName(def configuration, def architecture, def os, def scenario,
     // Need to change around some systems and other builds to pick up the right builds
     // to do that.
 
+    def isDefaultOrInnerloop = (scenario == 'default' || scenario == 'innerloop')
+
     def suffix = scenario != 'default' ? "_${scenario}" : '';
     if (isBuildOnly) {
         suffix += '_bld'
     }
     def baseName = ''
     switch (architecture) {
         case 'x64':
-            if (scenario == 'default') {
+            if (isDefaultOrInnerloop) {
```

Comment:
For all the new, innerloop jobs, I propose their names should include the architecture in all cases (which specifically means the x64 case), since there is no back compat issue.

Reply:
👍